pkgs/tools/virtualization/google-compute-guest-agent/0001-oslogin-add-disable-toggle-in-conf-file.patch

```diff
@@ -0,0 +1,40 @@
+From f318f525149b06365e9f0c7f642f09f401c3e1c1 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?F=C3=A9lix=20Baylac-Jacqu=C3=A9?= <felix@alternativebit.fr>
+Date: Fri, 5 Nov 2021 15:40:13 +0100
+Subject: [PATCH] oslogin: add disable toggle in conf file
```

Comment:
Please reword this patch - we should send this upstream, so it should be reworded a bit (to explain this is about NixOS, which doesn't want to use the imperative install/uninstall bits) - and maybe add some more comments to their code, so it's clear why the config boolean is read.

In another context, upstream seems to have been [open to accept patches](https://github.com/GoogleCloudPlatform/guest-oslogin/issues/33#issuecomment-832116232), as long as it doesn't impair other users - so a config option disabling the "installation" phases in their daemon might be accepted.

Reply:
@NinjaTrappeur are you fine with me trying to upstream this patch?

Reply:
:+1: sure!